src/main/1.3/scala/com/holdenkarau/spark/testing/StreamingSuiteCommon.scala

```diff
@@ -182,11 +180,15 @@ private[holdenkarau] trait StreamingSuiteCommon extends Logging with SparkContex
    * Returns a sequence of items for each RDD.
    */
   private[holdenkarau] def runStreams[V: ClassTag](
-      outputStream: TestOutputStream[V],
-      ssc: TestStreamingContext,
-      numBatches: Int,
-      numExpectedOutput: Int
-    ): Seq[Seq[V]] = {
+     outputStream: TestOutputStream[V],
+     ssc: TestStreamingContext,
+     numBatches: Int,
+     expectedOutput: Seq[Seq[V]],
```

Comment:
Why do we take this as a Seq if we are going to convert it to a mutable queue right away?